guides/plugins/plugins/administration/adding-responsive-behavior.md

```diff
@@ -0,0 +1,52 @@
+# Adding responsive behavior
+
+## Overview
+The Shopware 6 Administration provides two ways of adding classes to elements based on their size, the device helper and the `v-responsive` directive.
+Alternatively you can use `sccs` media queries to make your plugin responsive.
+Learn how to use `scss` here:
+
+{% page-ref page="./add-custom-styles.md" %}
+
+## DeviceHelper
+
+The DeviceHelper provides methods to get device and browser information like the current viewport size.
+The helper methods can be accessed with `this.$device` in every Vue component,
+since it is bound to the Vue prototype.
+
+It provides functionality to run a function on resize to for example add classes to a element.
```

Comment:
Please revisit this sentence. That's not proper English and I find it hard to suggest anything here, as I do not understand, what you want to say here